CHANGELOG.md

```diff
@@ -26,17 +30,17 @@ The major new feature in this release allows users to keep the aggregated
 data in a continuous aggregate while dropping the raw data with drop_chunks.
 This allows users to save storage by keeping only the aggregates.
 
-The semantics of the refresh_lag parameter for continuous aggregates has 
+The semantics of the refresh_lag parameter for continuous aggregates has
```

Comment:
```suggestion
The semantics of the `refresh_lag` parameter for continuous aggregates has
```

Reply:
I am going to opt out of making changelog content changes since this is not in scope of the PR.